keps/sig-storage/2268-non-graceful-shutdown/README.md

```diff
@@ -304,6 +332,8 @@ the health of the service?**
   - 99% percentile over day of absolute value from (job creation time minus expected
     job creation time) for cron job <= 10%
   - 99,9% of /health requests per day finish with 200 code
+  The failover should always happen if the feature gate is enabled, the taint
```

Comment:
I bet those controller metrics I mentioned above could be used here.

Reply:
Added.